roles/github/defaults/main.yml

```diff
@@ -131,6 +146,23 @@ github_run_infra_vm_service_deploy:
   arguments: "{{ github_kayobe_task_arguments }}"
   concurrency_group: infra
 
+github_octavia_certificates_schedule_trigger:
+  schedule:
+    cron: "{{ github_octavia_certificates_schedule }}"
+
+github_octavia_certificates_workflow_dispatch_trigger:
+  workflow_dispatch: "{{ github_octavia_certificates_input }}"
+
+github_run_kolla_ansible_octavia_certificates:
+  file_name: run-kolla-ansible-octavia-certificates-check-expiry.yml
+  use_bespoke: true
+  expiry_time: "{{ github_default_octavia_certificates_expiry_time }}"
+  trigger: >-
+    {{ github_octavia_certificates_schedule_trigger if
```

Comment:
Perhaps you'd still want workflow dispatch if you have a schedule?

Reply:
Yeah that would make sense, made the change but need to test still and see if my jinja knowledge is correct

Reply:
Fixed and done